properties.ini

```diff
@@ -0,0 +1,170 @@
+; this file is used to define all necessary options for virt-who testing
+; all default values are null, need to update according to the test requirements
+
+[host]
+; the rhel system to run virt-who service
+; virtwho/docker.py and kickstart.py can help to install a host
+ip=
+user=
+passwd=
+
+[repo]
+; the base url of the rhel compose for special cases
+rhel_base=
+
+[proxy]
+; a proxy server is necessary if run the related test cases
+server=
+port=
+
+[rhsm]
+; the datas of stage or product candlepin server
+server=
+user=
+passwd=
+org=
+
+[satellite]
+; the datas of satellite server
+; virtwho/satellite.py can help to deploy a satellite server
+server=
+admin_user=
+admin_passwd=
+default_org=
+default_env=
+extra_org=
+activation_key=
+
+[vcenter]
+; esx_ datas are for the esxi host which the guest is based on
+vcenter_ip=
+vcenter_admin_user=
+vcenter_admin_passwd=
+vcenter_ssh_user=
+vcenter_ssh_password=
```

Comment:
To match the configure options from /etc/virt-who.d:
#server=             ; insert hostname or ip address of the server to connect to
#username=        ; username for server authentication
#password=        ; password for server authentication
It's more better if we can have a uniform definition for all the hypervisors:
[vcenter]
server=
username=
password=
ssh_username=
ssh_password=
We will get the values as:  vcenter.server, vcenter.username, vcenter.password

Reply:
I updated it and reordered to put all esx_ options together, guest_ options together to make it clear.
```
[vcenter]
server=
username=
password=
ssh_username=
ssh_password=
esx_ip=
esx_username=
esx_password=
esx_uuid=
esx_hwuuid=
esx_hostname=
esx_version=
esx_state=
esx_cpu=
esx_cluster=
guest_name=
guest_ip=
guest_username=
guest_password=
guest_uuid=
guest_state=
```